core/src/main/scala/cats/syntax/all.scala

```diff
@@ -38,6 +38,7 @@ trait AllSyntax
     with MonadErrorSyntax
     with MonadSyntax
     with MonoidSyntax
+    with NestedSyntax
```

Comment:
unfortunately changing `AllSyntax` break binary compatibility on scala 2.11-. As a work around, please let the `AllSyntaxBinCompat1` above extend the new `NestedSyntax`